Utilities/aliceHLTwrapper/test/testMessageFormat.cxx

```diff
@@ -17,7 +17,7 @@
 #include "Headers/DataHeader.h"
 #include "Headers/HeartbeatFrame.h"
 
-namespace o2::AliceHLT {
+namespace o2 { namespace AliceHLT {
```

Comment:
Given the tests pass, I would say it's fine. ;-) If it's not fine, we should have the clang-tidy / clang-format to spot it. For sure the current code has other instances of the same kind and for sure the current code is violating our requirements in terms of C++ features being used.